conviva/README.md

```diff
@@ -26,26 +26,28 @@ This monitor requires:
 
 ### INSTALLATION
 
-Install the latest version of the SignalFx Smart Agent as described [here](https://github.com/signalfx/integrations/tree/master/signalfx-agent).
+Install the latest version of the SignalFx Smart Agent as described in [Installation](https://github.com/signalfx/integrations/tree/master/signalfx-agent).
 
 ### CONFIGURATION
 
-Find and edit the SignalFx Smart Agent configuration file `agent.yaml` to configure the conviva monitor as described <a target="_blank" href="https://github.com/signalfx/signalfx-agent/blob/master/docs/monitors/conviva.md">here</a>. Use the metric parameters <a target="_blank" href="https://github.com/signalfx/integrations/blob/master/conviva/docs/conviva_metrics.md">here</a> as values for `metricParameter`.
+1. Open the SignalFX Smart Agent `agent.yaml`configuration file.
+2. Edit the <a target="_blank" href="https://github.com/signalfx/signalfx-agent/blob/master/docs/monitors/conviva.md#overview">`monitors`</a> section.
```

Comment:
If you look at the screenshot, since **monitors** has the  ` ` symbols, it makes the hyperlink not overly obvious. 
<img width="1166" alt="Screen Shot 2020-05-21 at 10 50 59 AM" src="https://user-images.githubusercontent.com/54906773/82578227-bb954680-9b51-11ea-8a02-0500af47d907.png">

You can either remove the ` ` so that the text is in blue, making it obvious that it's a link, or you can keep the  ` `, and then add a bullet point below the line that says, "To learn how to edit this monitor, see [Conviva](* To learn how to edit this monitor, see [conviva][conviva.md ).)

(Also, in the story, I provided a link to GitHub; that was my mistake, I noticed that there is a link to the doc site, so we should use that link instead.) 

First link: https://docs.signalfx.com/en/latest/integrations/agent/monitors/conviva.html

Second link: https://docs.signalfx.com/en/latest/integrations/agent/monitors/conviva.html#conviva-monitor-metric-parameters-and-metrics